docker/scripts/install-docker-compose-ubuntu.sh

```diff
@@ -0,0 +1,28 @@
+# Ask for the user password
+# Script only works if sudo caches the password for a few minutes
+sudo true
+
+# Install kernel extra's to enable docker aufs support
+# sudo apt-get -y install linux-image-extra-$(uname -r)
+
+# Add Docker PPA and install latest version
+# sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 36A1D7869245C8950F966E92D8576A8BA88D21E9
+# sudo sh -c "echo deb https://get.docker.io/ubuntu docker main > /etc/apt/sources.list.d/docker.list"
+# sudo apt-get update
+# sudo apt-get install lxc-docker -y
+
+# Alternatively you can use the official docker install script
+wget -qO- https://get.docker.com/ | sh
+
+# Install docker-compose
+COMPOSE_VERSION=`git ls-remote https://github.com/docker/compose | grep refs/tags | grep -oE "[0-9]+\.[0-9][0-9]+\.[0-9]+$" | sort --version-sort | tail -n 1`
+sudo sh -c "curl -L https://github.com/docker/compose/releases/download/${COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose"
+sudo chmod +x /usr/local/bin/docker-compose
+sudo sh -c "curl -L https://raw.githubusercontent.com/docker/compose/${COMPOSE_VERSION}/contrib/completion/bash/docker-compose > /etc/bash_completion.d/docker-compose"
```

Comment:
Why not reusing the existing code?:

https://github.com/rhcs-dashboard/ceph-dev/blob/3aa188a5ed2efaa142b9bfe2088743a29b405e92/docker/scripts/install-docker-compose-centos-rhel.sh#L41-L45